packages/alfa-rules/src/sia-r54/rule.ts

```diff
@@ -0,0 +1,54 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { Node } from "@siteimprove/alfa-aria";
+
+import { isIgnored } from "../common/predicate";
+import { expectation } from "../common/expectation";
+
+const { and, not } = Predicate;
+const { isElement } = Element;
+
+export default Rule.Atomic.of<Page, Element>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r54",
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return document
+          .descendants({ flattened: true, nested: true })
+          .filter(isElement)
+          .filter(
+            and(not(isIgnored(device)), (element) =>
+              Node.from(element, device)
+                .attribute("aria-live")
+                .some((attribute) => attribute.value === "assertive")
+            )
+          );
+      },
+
+      expectations(target) {
+        return {
+          1: expectation(
+            Node.from(target, device)
+              .attribute("aria-atomic")
+              .some((attribute) => attribute.value === "true"),
+            () => Outcomes.IsAssertive,
+            () => Outcomes.IsNotAssertive
+          ),
+        };
+      },
+    };
+  },
+});
+
+export namespace Outcomes {
+  export const IsAssertive = Ok.of(
```

Comment:
Actually, the name should be `isAtomic` since this is what the expectation check.